packages/sirius-web/backend/sirius-web-infrastructure/src/main/java/org/eclipse/sirius/web/infrastructure/graphql/GraphQLConfiguration.java

```diff
@@ -74,18 +75,20 @@ public GraphQL graphQL(GraphQLSchema graphQLSchema) {
     }
 
     @Bean
-    public GraphQLSchema graphQLSchema(ResourcePatternResolver resourcePatternResolver, GraphQLWiringFactory graphQLWiringFactory, List<IDataFetcherWithFieldCoordinates<?>> dataFetchersWithCoordinates) {
+    public GraphQLSchema graphQLSchema(ResourcePatternResolver resourcePatternResolver, GraphQLWiringFactory graphQLWiringFactory, List<IDataFetcherWithFieldCoordinates<?>> dataFetchersWithCoordinates, Optional<IGraphQLCodeRegistryTransformer> transformer) {
```

Comment:
I'll just update this part in order to allow the support for multiple transformers. I know from experience that we will end up with multiple project trying to use this API at the same time.